itests/hive-unit/src/test/java/org/apache/hive/jdbc/AbstractTestJdbcGenericUDTFGetSplits.java

```diff
@@ -179,15 +176,9 @@ protected void testGenericUDTFOrderBySplitCount1(String udtfName, int[] expected
     query = "select " + udtfName + "(" + "'select value from " + tableName + " where value is not null limit 2', 5)";
     runQuery(query, getConfigs(), expectedCounts[5]);
 
-    query = "select " + udtfName + "(" +
-        "'select `value` from (select value from " + tableName + " where value is not null order by value) as t', 5)";
+    query = "select " + udtfName + "(" + "'select `value` from (select value from " + tableName +
+        " where value is not null order by value) as t', 5)";
     runQuery(query, getConfigs(), expectedCounts[6]);
-
-    List<String> setCmds = getConfigs();
-    setCmds.add("set hive.llap.external.splits.order.by.force.single.split=false");
-    query = "select " + udtfName + "(" +
-        "'select `value` from (select value from " + tableName + " where value is not null order by value) as t', 10)";
-    runQuery(query, setCmds, expectedCounts[7]);
```

Comment:
testGenericUDTFOrderBySplitCount1 now uses only 6 values from expectedCounts.... update the test inputs

Reply:
Done, there were 8 values before. I have updated the expected count to use 7 values not. 
https://github.com/apache/hive/pull/4249/files#diff-47b7f6f11c3aabbf6811b6d4d2e48162b68e4d9f30241b956e17b0d3960ec223L43